src/shared/scrapers/USA/CA/arcgis-counties

```diff
@@ -0,0 +1,99 @@
+import * as fetch from '../../../lib/fetch/index.js';
+import * as parse from '../../../lib/parse.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography/index.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+function addCounty(county) {
+  return county + ' County';
+}
+
+const scraper = {
+  country: 'USA',
+  state: 'CA',
+  priority: 0,
+  url: 'https://opendata.arcgis.com/datasets/06fcfe6147574a75afea97a1f7565dc7_0.csv',
+  aggregate: 'county',
```

Comment:
Can we cite the source with a human-readable name with `sources: [{name: 'Whatever', url: 'humanreadibleurl' }]`?